Resources/Prototypes/Entities/Structures/Furniture/seats.yml

```diff
@@ -201,6 +201,9 @@
           MaterialWoodPlank:
             min: 1
             max: 1
+  - type: Tag
+    tags:
+    - Wooden
```

Comment:
I guess you might as well also add the wooden tag to some other wooden objects.
Just based on ctrl+f "woodhit.ogg" and "MaterialWoodPlank", there is `Bookshelf`, `TableWood`, and `TableCarpet`.

Reply:
sure, i'll add it to those as well